src/components/Palette.tsx

```diff
@@ -0,0 +1,44 @@
+// This component holds the shapes
+
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import { Shape } from "./Shape";
+
+export interface PaletteProps {
+
+}
+
+export class Palette extends React.Component<PaletteProps, {}> {
+    render() {
+        return <div className = "palette">
+                <div className = "shape-wrapper">
+                    <Shape kind = "square">                  
+                        <svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                            <defs></defs>
+                            <rect id="Rectangle-85" stroke="none" fill="#465659" fill-rule="evenodd" x="0" y="0" width="80" height="80"></rect>
```

Comment:
You'll probably want to reuse these shapes in your canvas as well. You could change the structure around a little bit, by having `<Rect>`, `<Triangle>` and `<Circle>` components that render each shape. Then you could reuse them in your palette by doing:

```ts
<div className="palette">
  <ul>
    <li><svg><Rect/></svg></li>
    <li><svg><Triangle/></svg></li>
    <li><svg><Circle/></svg></li>
  </ul>
</div>
```

Probably needs some tweaking, but you get the idea.